Gemfile

```diff
@@ -51,6 +50,14 @@ gem 'sidekiq-statsd', '0.1.5'
 
 gem 'deprecated_columns', '0.1.0'
 
+if ENV['GDS_API_ADAPTERS_DEV']
```

Comment:
The git ref is only temporary. I'll remove that commit before merging. Is it the ENV variable that you're not happy with. It just seemed like a good idea as there was prior art in the file as it seemed likely be making changes to the `gds-api-adapter` repo as we migrate and it makes for easier concurrent development. Happy to change if you disagree.